cmd/zpool/zpool_main.c

```diff
@@ -1568,10 +1563,10 @@ zpool_do_create(int argc, char **argv)
 		goto errout;
 
 	/*
-	 * We dont' store the creds as a normal property, so remove
+	 * We dont' store the creds profile as a normal property, so remove
```

Comment:
[nit while you're here]
```suggestion
	 * We don't store the creds profile as a normal property, so remove
```

Reply:
done